frontend/components/Meetups/useSortMeetups.js

```diff
@@ -0,0 +1,55 @@
+export function useSortMeetups (selectedMeetups, sortType) {
```

Comment:
I think I would not like to have non-components in components directory. Could you move it somewhere else? `Meetup/useFindNextMeetup.js` too.

Reply:
moved :)